scripts/irods/test/resource_suite.py

```diff
@@ -1167,3 +1167,96 @@ def test_itrim_no_op(self):
 
         # try to trim down to repl_count
         self.admin.assert_icommand("itrim -N {repl_count} {filename}".format(**locals()), 'STDOUT_SINGLELINE', "Total size trimmed = 0.000 MB. Number of files trimmed = 0.")
+
+
+    ###################
+    # detached mode
+    ###################
+    def test_detached_mode(self):
+
+        try:
+            file1 = "f1"
+            file2 = "f2"
+            resource_host = "irods.org"
+            resource_name = 'detached_resource'
+
```

Comment:
Why an empty line here? The next line seems like it can be grouped with the ones above.

Same thing applies to other tests.

Reply:
done